src/styles/main.scss

```diff
@@ -115,6 +115,15 @@ header.navbar .navbar-brand {
   display: block !important;
 }
 
+.cke_combo {
```

Comment:
Just realized it can be just in one line `.cke_combo .cke_combo_button .cke_combo_text {`